src/debugger.ts

```diff
@@ -95,6 +106,11 @@ export namespace Debugger {
      */
     readonly service: DebuggerService;
 
+    /**
+     * The body of widget.
+     */
+    readonly body: SplitPanel;
```

Comment:
Yeah, actually I think I agree with @jtpio ... this should probably just be `const body = new SplitPanel()` as well ... it doesn't need to be public.

In some future PR would should probably store the relative sizes of the split panel for restoring state on refresh (but not now).

Reply:
> In some future PR would should probably store the relative sizes of the split panel for restoring state on refresh (but not now).

That would definitely be useful :+1: 
Especially if a user wants to collapse some of the sections in the sidebar: https://github.com/jupyterlab/debugger/issues/325